pkg/webui/console/views/application-overview/application-overview.styl

```
@@ -12,6 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+.title-section
+  border-normal('bottom')
+  margin-bottom: $ls.s
```

```suggestion
  +media-query-min($bp.s)
    margin-bottom: $ls.s
```

Better to avoid overwrites and apply responsive styles in its necessary scope.